odxtools/database.py

```diff
@@ -139,6 +140,16 @@ def refresh(self) -> None:
         for dlc in self.diag_layer_containers:
             dlc._resolve_odxlinks(self._odxlinks)
 
+        # resolve short name references for containers which do not do
+        # inheritance (we can call directly call _resolve_snrefs())
+        context = SnRefContext()
+        context.database = self
+
+        for subset in self.comparam_subsets:
+            subset._resolve_snrefs(context)
+        for spec in self.comparam_specs:
+            spec._resolve_snrefs(context)
+
```

Comment:
tangentially (as usual): while working on this, I noticed that in the current `main` version `._resolve_snrefs()` is not called for `ComparamSpec` and `ComparamSubset`. Also as usual I forgot that this was in the changes later. I moved it to a separate commit and if you want, I can move it to a separate PR, but IMO that would be a bit too heavy-handed...

(I force pushed this because splitting a patch into two does not work too well with fixups. sorry...)